driver/web/rest/adminapis.go

```diff
@@ -1387,3 +1387,460 @@ func (h AdminApisHandler) GetContentItemsCategories(claims *tokenauth.Claims, w
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	w.Write(jsonData)
+}
+
+// GetDataContentItem Gets a data content type item
+// @Description Gets a data content type item
+// @Tags Admin
+// @ID AdminGetDataContentItem
+// @Accept json
+// @Produce json
+// @Success 200
+// @Security AdminUserAuth
+// @Router /admin/data/{key} [get]
+func (h AdminApisHandler) GetDataContentItem(claims *tokenauth.Claims, w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	key := vars["key"]
+
+	resData, err := h.app.Services.GetDataContentItem(claims, key)
+	if err != nil {
+		log.Printf("Error on getting data content type with id - %s\n %s", key, err)
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	data, err := json.Marshal(resData)
+	if err != nil {
+		log.Println("Error on marshal of data content type")
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	w.Write(data)
+}
+
+// GetDataContentItems Gets data content items
+// @Descriptions Gets data content items
+// @Tags Admin
+// @ID AdminGetDataContentItems
+// @Param category body string false "category - get all data content items based on category"
+// @Accept json
+// @Produce json
+// @Success 200
+// @Security AdminUserAuth
+// @Router /admin/data [get]
+func (h AdminApisHandler) GetDataContentItems(claims *tokenauth.Claims, w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	key := vars["id"]
+
+	category := r.FormValue("category")
+	if len(category) <= 0 {
+		log.Print("Missing category\n")
+		http.Error(w, "missing 'catgory' form param", http.StatusBadRequest)
+		return
+	}
+
+	resData, err := h.app.Services.GetDataContentItems(claims, category)
+	if err != nil {
+		log.Printf("Error on getting data content type with id - %s\n %s", key, err)
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	data, err := json.Marshal(resData)
+	if err != nil {
+		log.Println("Error on marshal of data content type")
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	w.Write(data)
+}
+
+// UpdateDataContentItem Updates a content item with the specified id.
+// @Description Updates a content item with the specified id.
+// @Tags Admin
+// @ID AdminUpdateDataContentItem
+// @Accept json
+// @Produce json
+// @Success 200 {object} model.DataContentItem
+// @Security AdminUserAuth
+// @Router /admin/data/{id} [put]
+func (h AdminApisHandler) UpdateDataContentItem(claims *tokenauth.Claims, w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	id := vars["id"]
```

Comment:
This path parameter doesn't match up with the docs for this API and it isn't used except to log an error.